vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_value_util.cc

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_value_util.h"
```

Comment:
See previous comment about filename matching the object name, so here that would be `embedding_pipeline_value_util`